src/pages/[boardId]/PaperThumbnail.tsx

```
@@ -5,18 +5,26 @@
~import {css} from '@emotion/react';
 import {MultiplePaper} from '@type/model';
 
 import {useTheme} from '@/theme/DesignProvider';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)